src/es/ikigaimangas/src/eu/kanade/tachiyomi/extension/es/ikigaimangas/IkigaiMangas.kt

```diff
@@ -222,7 +299,7 @@ class IkigaiMangas : HttpSource(), ConfigurableSource {
     }
 
     override fun pageListRequest(chapter: SChapter): Request =
-        GET(fetchedDomainUrl + chapter.url.substringBefore("#"), lazyHeaders)
+        GET(fetchedDomainUrl + chapter.url.substringBefore("#"), headers)
```

Comment:
Maybe use `HttpUrl` to remove fragment.

Reply:
Probably. Other half of the extension is also working directly with strings, so it's a bigger refactor.